CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Changelog
 
+### 2.79
```

Comment:
You'll actually be releasing 2.80. Sorry for the confusion, now that we aren't using the wiki for changelogs, I keep forgetting that we need to add release notes for security fixes here ourselves! Could you add this content for 2.79?

```
### 2.79

Release date: 2020-02-12

* Security: Fix sandbox bypass vulnerability. ([SECURITY-1710](https://jenkins.io/security/advisory/2020-02-12/#SECURITY-1710))
```

Reply:
addressed in 092c181